### ISSUE TITLE
upload to shinyapps.io

### ISSUE THREAD

**Issue by yenzichun** [jbkunst/highcharter#192]
Hi, I am new to use highcharter, and this is really a convenient tool.
However, when I was trying to publish my shiny app to shinyapps.io, it showed "ERROR: there is no package called ‘highcharter’" on the browser. But I can run the app on the local side.

Can you help me with this?
Thank you in advance.

**Comment by jbkunst** [comment]
GHi @yenzichun ,

Not sure what is going on. Have you tried to reinstall the packages?

**Comment by jbkunst** [comment]
Can be this same issue as https://github.com/rstudio/shinyapps/issues/73

Use require instead of library.

**Comment by yenzichun** [comment]
Hi Joshua,

Thanks for pointing out the possible solution. 
After I changed it to "require("highcharter")", the same error message did not show anymore.
But it showed "there's no function called highchartOutput()" instead, along with some warning messages:
![image](https://cloud.githubusercontent.com/assets/6956697/18761590/1c180728-8138-11e6-9e06-98339f59692e.png)

And as I dig deeper, I find out that this might be due to the encoding issue.

For example, when I run the app using the "Run App" in RStudio, the result is fine:
![image](https://cloud.githubusercontent.com/assets/6956697/18761716/aee1ab40-8138-11e6-902a-ff572845be8d.png)

However, when I run it from the shinyapps.io, it looks like this:
![image](https://cloud.githubusercontent.com/assets/6956697/18761777/e306af24-8138-11e6-93cb-1d4bc3aff185.png)

The words showing on X-axis is Chinese. But I have set my default file encoding to UTF-8.
Appreciate your help!!

**Comment by jbkunst** [comment]
Mmm.. not sure how treat this issue. May be you can mention some Chinese highcharter users in this issue , like @shrektan

**Comment by jbkunst** [comment]
#50

**Comment by yenzichun** [comment]
Hi Joshua,

After spending some time looking for solution, I found [this](http://shiny.rstudio.com/articles/unicode.html).
So I realized that although my app.R file is in UTF-8, the .rda object which I loaded into app.R file is in **unicode** format.

Here's what I did:
```{r}
load(file = "data.rda") # a list object called input_lst
enc2native(input_lst)
Encoding(input_lst) <- "CP950"
input_lst <- iconv(input_lst, "CP950", "UTF-8")
```

I am not sure if that is the right way to deal with the issue, at least this time it works. Any thoughts would be appreciated.
Thank you!!
